sys/sys/param.h

```diff
@@ -192,8 +192,8 @@
 /*
  * Mach derived conversion macros
  */
-#define	round_page(x)	(((x) + PAGE_MASK) & ~(PAGE_MASK))
-#define	trunc_page(x)	((x) & ~(PAGE_MASK))
+#define	round_page(x)	__builtin_align_up((x), PAGE_SIZE)
+#define	trunc_page(x)	__builtin_align_down((x), PAGE_SIZE)
```

Comment:
```suggestion
#define	round_page(x)	roundup2(x, PAGE_SIZE)
#define	trunc_page(x)	rounddown2(x, PAGE_SIZE)
```

then fewer places need to actually implement it (which is important if you want to support GCC / older compilers).